tests/Solid/input

```diff
@@ -103,9 +103,38 @@
 #@  args = model1.C12=1.221
 #@  args = model1.C44=0.757
 #@  ignore = model1.E model1.nu
+#@
```

Comment:
There is a lot of extra whitespace here along with the original todo comments. Please clean this up.